child_compassion/wizard/delegate_child_wizard.py

```diff
@@ -52,11 +58,22 @@ def delegate(self, cr, uid, ids, context=None):
         child_ids = self._default_child_ids(cr, uid, context)
         child_obj = self.pool.get('compassion.child')
 
+        if wizard.date_end_delegation:
+            if datetime.strptime(wizard.date_delegation, DF) > \
+               datetime.strptime(wizard.date_end_delegation, DF):
+                raise orm.except_orm("Invalid value", "End date must "

```

Comment:
Put the error message inside _() function, so that it can be translated in French if needed.